src/main/java/com/minecolonies/client/render/EmptyTileEntitySpecialRenderer.java

```diff
@@ -1,7 +1,14 @@
 package com.minecolonies.client.render;
 
+import com.minecolonies.colony.buildings.BuildingTownHall;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.colony.buildings.BuildingTownHall'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)